src/backend/joanie/core/utils/contract_definition.py

```diff
@@ -38,9 +39,19 @@ def generate_document_context(contract_definition, user, order=None):
 
     address = AddressSerializer(user_address).data
 
+    try:
+        site_config = Site.objects.get_current().site_config
```

Comment:
We have probably to discuss about that because if we start using multiple site this will always return the site define by the `SITE_ID` settings.

Reply:
To resume our discussion, we decide to first consider the platform as mono site.